scripts/inject-github-link.js

```diff
@@ -0,0 +1,29 @@
+import { JSDOM } from 'jsdom';
+import jQuery from 'jquery';
+import relative from 'relative';
+
+export function injectGithubLink (html, filePath) {
+  const relativeLink = relative(process.cwd() + '/docs', filePath);
+  const paths = relativeLink.split('\/');
+
+  let githubLink;
+  if (paths[1] === 'commands') {
```

Comment:
should we check the length of paths before?

Reply:
It shouldn't matter. If `paths.length` is 0 or 1, `paths[1]` remains valid and will never equal `'commands'`. In the `else` block `paths.slice(0, paths.length - 1)` would be `[]` for lengths under 2.